src/sourcemaps/sourcemaps-wizard.ts

```diff
@@ -41,31 +46,37 @@ export async function runSourcemapsWizard(
 
   const selectedTool = await askForUsedBundlerTool();
 
-  // TODO: Add configuration instructions here, yada yada
-  // eslint-disable-next-line no-console
-  console.log('Great, I got everything I need for now:', {
-    sentryUrl,
+  await configureSelectedTool(selectedTool, {
```

Comment:
```suggestion
  await toolSetupFlow(selectedTool, {
```

l: Took me a sec to understand what is meant by "configureSelectedTool".